test/tbb/test_parallel_phase.cpp

```diff
@@ -69,7 +68,11 @@ std::size_t measure_median_start_time(tbb::task_arena* ta, const F1& start = F1{
     auto work = [&] {
         auto start_time = std::chrono::steady_clock::now();
         start();
-        tbb::parallel_for(std::size_t(0), num_threads, measure_start_time, tbb::static_partitioner{});
+        for(std::size_t thr = 0; thr < num_threads-1; ++thr) {
+            tbb::this_task_arena::enqueue(measure_start_time);
+        }
+        start_times[tbb::this_task_arena::current_thread_index()] = std::chrono::steady_clock::now();
+        barrier.wait();
```

Comment:
Consider adding comment why enqueueing is preferrable here. Perhaps, something like _Enqueueing the work to increase probability all the threads could find the work_ might suffice.